content/build/hydrobuild.md

```diff
@@ -451,3 +451,24 @@ default `docker` builder, run the following command:
 ```console
 $ docker context use default
 ```
+
+### How do I manage the build cache with Hydrobuild?
+
+You don't need to manage the builder's cache manually. The system manages it
+for you through [garbage collection](./cache/garbage-collection.md).
+
+Hydrobuild uses the following garbage collection limits:
+
+- 90% of 1TB of data
```

Comment:
@silvin-lubecki Is 1TB the current storage limit for GA or just EAP? I don't remember where we say 1TB was fine. That looks like a lot to me imo. I think 100GB or even 60GB would be enough. Ofc this base storage (c/s)hould be increased depending on the number of projects or users in the org. Would say 15% increase for each additional user (to be taken with a pinch of salt :sweat_smile:).

Reply:
Yes that is only for EAP right now, to get the max throughput from EBS volumes. Cc @macInfinity